frontend/src/app/testQueue/AoEForm/SymptomInputs.tsx

```diff
@@ -34,6 +37,9 @@ const SymptomInputs: React.FC<Props> = ({
 
   symptomOnsetRef,
 }) => {
+  function setFormattedOnsetDate(input = "") {
+    return setOnsetDate(formatDate(input));
+  }
```

Comment:
this is the second time we have this function. Maybe we should be changing `formatDate` to handle null values

Reply:
See https://github.com/CDCgov/prime-simplereport/pull/1820#discussion_r652171067.